### PR TITLE
20339 ensure no entities

### PR DESCRIPTION
**Pull request by NissimHadar**
https://highfidelity.fogbugz.com/f/cases/20339/
# Description
Adds a script to delete all entities around the avatar.  This solution was chosen over hiding and the re-showing the entities for 2 reasons:
1.  There may be invisible entities - these then would become visible after the test
2.  This complicates nitpick even more.
# Testing
1.  Run Interface and go to localhost.
2.  Move to a remote region (such as 9999,9999,9999)
3.  Create some entities at that location - (i.e. cube, sphere and text)
4.  Run the following test: https://raw.githubusercontent.com/NissimHadar/hifi_tests/20339-ensureNoEntities/tests/utils/deleteNearbyEntities.js
5.  Verify the added entities have been deleted.  There should be no entities nearby after the test has been completed.

### CODE REVIEW

File: tests/utils/deleteNearbyEntities.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// Remove all entities near the avatar
+const TEST_RADIUS = 1000;
+var entitiesToDelete = Entities.findEntities(MyAvatar.position, TEST_RADIUS);
```
Comment (SamGondelman) [inline tests/utils/deleteNearbyEntities.js]: I wonder if it wouldn't be better to just delete the entities in view, which I think you can do with `Entities.findEntitiesInFrustum(Camera.frustum)`.

**Comment by NissimHadar** [comment]
That would be risky.  Some tests rotate the viewpoint during the test.  One
that does that for sure is the camera test that tests camera rotation…

Remember that this only done during automatic testing.  There shouldn't be
any entities at all.  But, especially during testing, it has happened.
Adam has seen strange results sometimes, and this seems to be the problem.

File: tests/utils/deleteNearbyEntities.js (hunk 1 above)
Comment (NissimHadar) [reply tests/utils/deleteNearbyEntities.js]: That would be risky.  Some tests rotate the viewpoint during the test.  One
that does that for sure is the camera test that tests camera rotation…

Remember that this only done during automatic testing.  There shouldn't be
any entities at all.  But, especially during testing, it has happened.
Adam has seen strange results sometimes, and this seems to be the problem.

Comment (SamGondelman) [reply tests/utils/deleteNearbyEntities.js]: ah ok good point

**Pull request merged**